frontend/components/EventPage/TimeLeft.tsx

```diff
@@ -0,0 +1,21 @@
+
+import { ReactElement } from 'react'
+import styled from 'styled-components'
+import TimeAgo  from 'react-timeago';
+
+import { LIGHT_GRAY } from '../../constants'
+
+const TimeLeft = ({
+    start,
+    className,
+  }: {
+    start: Date
+    className?: string
+  }): ReactElement => (
+    <TimeAgo className ={className} date={start}/>
+  )
+  
+  export default styled(TimeLeft)`
```

Comment:
You should use `s` instead of `styled`. The rest of the codebase uses this convention, simply because it is a bit shorter to type out.